api/src/lib.rs

```diff
@@ -972,6 +999,20 @@ impl Endpoints {
             })
     }
 
+    fn comment_audio(&self, source_id: &SourceId, comment_id: &CommentId) -> Result<Url> {
+        self.base
+            .join(&format!(
+                "/api/_private/sources/id:{}/comments/{}/audio",
+                source_id.0, comment_id.0
+            ))
+            .chain_err(|| ErrorKind::Unknown {
+                message: format!(
+                    "Could not build sync URL for source id `{}` and comment id `{}`.",
```

Comment:
```suggestion
                    "Could not build put audio URL for source id `{}` and comment id `{}`.",
```

Reply:
Well spotted -- I fixed it, without the mention of "put" as this is just the url for the audio content -- it can also be get